HalloWing_Jump_Scare_Trap/hallowing_jump_scare_trap.py

```diff
@@ -0,0 +1,148 @@
+# HalloWing Jump Scare Trap
+# use PIR sensor, speaker, and servo
+import time
+import array
+import math
+import board
+import displayio
+import pulseio
+from adafruit_motor import servo
+import digitalio
+import touchio
+import audioio
+# Setup LED and PIR pins
+LED_PIN = board.D13  # Pin number for the board's built in LED.
+PIR_PIN = board.SENSE   # Pin port connected to PIR sensor output wire.
+# Setup digital input for PIR sensor:
+pir = digitalio.DigitalInOut(PIR_PIN)
+pir.direction = digitalio.Direction.INPUT
+# Setup digital output for LED:
+led = digitalio.DigitalInOut(LED_PIN)
+led.direction = digitalio.Direction.OUTPUT
+# Setup servo
+# servo = pulseio.PWMOut(board.D4, frequency=50)
+pwm = pulseio.PWMOut(board.D4, duty_cycle=2 ** 15, frequency=50)
+servo = servo.Servo(pwm)
+
+# Setup cap touch button
+ready_button = touchio.TouchIn(board.TOUCH1)
+
+def servo_ready():
+    servo.angle = 0
+def servo_release():
+    servo.angle = 90
+
+# Set servo to ready position
+servo_ready()
+# Function for playing wav file, releasing servo
+def play_wave():
+    wave_file = open("hiss01.wav", "rb")  # open a wav file
+    wave = audioio.WaveFile(wave_file)
+    audio.play(wave)  # play the wave file
+    while audio.playing:  # turn on LED, turn servo
```

Comment:
i think you want to move these three lines above the while loop, and then just 'pass' within the loop, not continuously print out Motion Detected right?

Reply:
Perfect, this worked great.

Reply:
did you actually commit this change?

Reply:
No, not yet, I was waiting on a decision on the tone A0 thing. Should I not have waited on that? Looks like that one was removed?

Reply:
i dont know how you saved your files but please commit it and push so we can wrap this up

Reply:
I just commited and pushed.